user_guide/data_management.rst

```diff
@@ -0,0 +1,234 @@
+.. _user-guide-data-management-basics:
+
+Data Management Basics
+======================
+
+With OroCRM, you can record, store, process and analyze various customer-related information. This information may cover 
+different areas including personal and contact details, records of the customer activity, possible sales and 
+arrangements, and many others. This can be any type of information necessary to monitor, manage and understand specific 
+customer-related activities. 
+
+In this article we will overview the ways data is represented and managed in OroCRM. For 
+each point that requires further explanation, you will find links to related documentation. 
```

Comment:
Edit: In this article, we will review the ways that data is represented and managed in OroCRM. For each point that requires further explanation, you will find links to related documentation.